publisher-sdk/src/main/java/com/criteo/publisher/logging/RemoteLogRecords.kt

```diff
@@ -52,6 +53,16 @@ data class RemoteLogRecords(
     ERROR,
 
     @SerializedName("None")
-    NONE
+    NONE;
+
+    companion object {
+      fun fromAndroidLogLevel(logLevel: Int): RemoteLogLevel? = when (logLevel) {
+        Log.DEBUG -> DEBUG
+        Log.INFO -> INFO
+        Log.WARN -> WARNING
+        Log.ERROR -> ERROR
+        else -> null
```

Comment:
The `RemoteLogLevel` correspond to what is accepted by the API. So `null` happens if someone uses a log level that is not supported by the API.

The remaining Android log levels that are mapped to null are:
- `VERBOSE`
- `ASSERT` (but there is no `Log.a` method, so I doubt it will be used)

I don't think the API will handle such levels. But I would not say that it is not desirable. Conversely, the `VERBOSE` might be used for very verbose logs that we don't want to export to API. And logging a warning in case of `null` is a bad idea IMOH: it will generate a remote logs for every verbose logs, which is quite the opposite of the expected behavior.

Reply:
ok sounds good